### PR TITLE
New script for PromethION data transfer

### PR DESCRIPTION
**Pull request by ssjunnebo**
First draft for a script to transfer data from the promethion to ngi-nas. To be run hourly as  cronjob on the promethion. Any feedback appreciated!

### CODE REVIEW

File: promethion_transfer.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+""" Transfers new PromethION runs to ngi-nas using rsync.
```
Comment (aanil) [inline promethion_transfer.py:1]: Should this be a part of TACA? The Illumina runs are transferred with lsync, so I'm not quite sure where this should go.

Comment (alneberg) [reply promethion_transfer.py:1]: I think the plan was first to have it as a cronjob but we needed some additional logic (keeping track of which sync is the last) so it made sense to make it a python script. I think TACA is a good place to keep it yes

Comment (ssjunnebo) [reply promethion_transfer.py:1]: Sure, I can move it to TACA instead 👍

**Comment by ssjunnebo** [comment]
Moving script to TACA instead

File: promethion_transfer.py (hunk 1 above)
Comment (alneberg) [reply promethion_transfer.py:1]: Oh, no I'm sorry! I thought this PR already was on TACA... I remember now that we discussed that we shouldn't install more than absolutely necessary on the PromethION. Again, sorry for the mess and all the extra work

**Pull request closed without merging**